src/index.ts

```diff
@@ -1,3 +1,5 @@
+process.env.NODE_ENV = process.env.NODE_ENV || 'development'
```

Comment:
Most production environments will explicitly set the `NODE_ENV`  to `production` so `development` as a fallback is an easy way to ensure that local installations have a sensible value for this.